lib/plugins/helper/url_for.js

```diff
@@ -22,7 +23,7 @@ function urlForHelper(path = '/', options) {
 
   // Resolve relative url
   if (options.relative) {
-    return this.relative_url(this.path, path);
+    return relative_url(this.path, path);
```

Comment:
this fix is already present in https://github.com/hexojs/hexo/pull/3710/files#diff-d89aa048c56f2709ea59155d3f91aa81 , I put this as not to depend #3710